clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/agent/KubernetesInstanceCachingAgent.groovy

```diff
@@ -34,6 +34,8 @@ import static com.netflix.spinnaker.cats.agent.AgentDataType.Authority.AUTHORITA
 
 @Slf4j
 class KubernetesInstanceCachingAgent extends KubernetesCachingAgent {
+  static final String CACHE_TTL_ANNOTATION = "spinnaker.cache-ttl"
```

Comment:
A small nit, but I'd like to make the annotations match k8s' designated annotations: https://kubernetes.io/docs/reference/labels-annotations-taints/

I think something like "cache.spinnaker.io/ttl" could make sense